packages/superset-ui-chart-controls/src/utils/getColorFormatters.ts

```diff
@@ -30,19 +30,23 @@ export const round = (num: number, precision = 0) =>
 export const rgbToRgba = (rgb: string, alpha: number) =>
   rgb.replace(/rgb/i, 'rgba').replace(/\)/i, `,${alpha})`);
 
-export const getOpacity = (value: number, cutoffPoint: number, extremeValue: number) => {
-  const MIN_OPACITY = 0.3;
-  const MAX_OPACITY = 1;
-
-  return extremeValue === cutoffPoint
-    ? MAX_OPACITY
+const MIN_OPACITY = 0.05;
```

Comment:
Should we have separate `MIN_OPACITY_BOUNDED = 0.05` and `MIN_OPACITY_UNBOUNDED = 0`?

Reply:
Done